iOS/MMIWG2S/ViewControllers/PageViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  PageViewController.swift
+//  MMIWG2S
+//
+//  Created by Mark Amaro on 8/21/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+import UIKit
+import ARKit
+
+protocol NextPageDelegate {
+    func nextSlide(_ vcFrom: Int)
+}
+
+class PageViewController: UIPageViewController, UIPageViewControllerDataSource, NextPageDelegate {
+    
+    private var array: [IntroViewController] = [IntroViewController(background: UIImage(named: "slide1") ?? UIImage(), gradient: UIImage(named: "gradient1.png") ?? UIImage(), mmiwtext:"slideview1.text", page: 0), IntroViewController(background: UIImage(named: "slide2") ?? UIImage(), gradient: UIImage(named: "gradient2.png") ?? UIImage(), mmiwtext:"slideview2.text", page: 1), IntroViewController(background: UIImage(named: "slide3") ?? UIImage(), gradient: UIImage(named: "gradient3.png") ?? UIImage(), mmiwtext:"slideview3.text", page: 2) , IntroViewController(background: UIImage(named: "slide4") ?? UIImage(), gradient: UIImage(named: "gradient4.png") ?? UIImage(), mmiwtext:"slideview4.text", page: 3)]
```

Comment:
This works! Putting the individual array entries on separate lines could be helpful for readability. Either way